index.js

```diff
@@ -365,7 +366,15 @@ module.exports = class Combobox {
     this.currentOption = this.currentOpts[option];
     // show pseudo focus styles
     this.pseudoFocus();
-    if (fromKey) { this.ensureVisible(); }
+    // Dectecting if element is inView and scroll to it.
+    this.currentOpts.forEach((opt) => {
+      if (!isElementInView(opt, this.list).top) {
+        if (opt.classList.contains('active')) {
+          const activeNode = document.querySelector('.active');
```

Comment:
this will break as soon as there are 2 comboboxes on the page...didn't we just confirm that opt has active - meaning thats what this combobox's activeNode is??